libqtile/widget/image.py

```diff
@@ -58,45 +58,26 @@ def _configure(self, qtile, bar):
             raise ValueError("Filename not set!")
 
         self.filename = os.path.expanduser(self.filename)
```

Comment:
I feel like there should be a check that the file exists and log it and exit nicely if it doesn't, I don't know if that should be in the `Image.from_path`, or before that.